addons/rating/models/rating_mixin.py

```diff
@@ -193,7 +193,7 @@ def rating_apply(self, rate, token=None, feedback=None, subtype_xmlid=None):
             if hasattr(self, 'message_post'):
                 feedback = tools.plaintext2html(feedback or '')
                 self.message_post(
-                    body="<img src='/rating/static/src/img/rating_%s.png' alt=':%s/10' style='width:18px;height:18px;float:left;margin-right: 5px;'/>%s"
+                    body="<span style='float:left;margin-right: 5px;'>Rating:</span><img src='/rating/static/src/img/rating_%s.png' alt=':%s/10' style='width:18px;height:18px;float:left;margin-right: 5px;'/>%s"
```

Comment:
Hello, as I mentioned, this change is not ok. We need to fix the compute of "is empty" to take into account messages with only images instead.

Current diff will not fix issue with existing rating messages, and it will also not fix the issue for other messages that are just an emoji or an image.

Reply:
Hello, yes you are right but now the SPEC is updated and as per updated SPEC we need to display like ` Rating: emoji(<image>)` instead of simple emoji.

Reply:
Ok I didn't see that part. But then I guess the new text will need to be translatable, and be changed only in master. + I would like to double check that style while we are at it, I don't understand why using float in particular.

Reply:
If we remove 'float' style on span tag the emoji will display before the text "Rating:".

Reply:
But if we remove float also from the emoji tag?

Reply:
Then feedback description will be displayed on the new line separately. Not in the same line just like in old behaviour.

Reply:
Ok thank you for checking, let's keep this float for now then, it's out of scope of current task.

Reply:
So just still need to make the "Rating" text translatable I guess.